### PR TITLE
Support more than 32-bits of modes using MSP_ACTIVEBOXES API

### PR DESCRIPTION
**Pull request by digitalentity**
Now we can support up to 52 box modes (20 more than now)

### CODE REVIEW

File: tabs/modes.js [hunk 1]
```diff
@@ -11,10 +11,6 @@ TABS.modes.initialize = function (callback) {
         googleAnalytics.sendAppView('Modes');
     }
 
-    function get_box_data() {
```
Comment (DzikuVx) [inline tabs/modes.js]: To be honest I would leave this message in place. periodicStatusUpdater can be postponed by serial port balancing and it's not guaranteed that active modes will be loaded when TAB is entered

Comment (digitalentity) [reply tabs/modes.js]: MSP_BOX was not implemented at all - it was reused as MSP_ACTIVEBOXES.

Comment (digitalentity) [reply tabs/modes.js]: I'll add it back

**Comment by digitalentity** [comment]
Should be ready now

**Pull request merged**